### DISCUSSION TITLE
404 on dynamic routing.

### DISCUSSION THREAD

**Discussion by ppot** [vercel/next.js#14571] [Help] [answered]
HI all,  as im working on my final engineering project i'm using NextJs. Yay 😄 . 

However I'm kind of stuck at the moment. I have a page under my teams 

```
Page
   - teams
         - [team].tsx
```

When using link component to navigation to my team dynamic routes. It work.
```
              <Link href="/teams/[team]" as={`/teams/${team.urlname}`}>
                <a><ChevronRight size={18} /></a>
              </Link>
```
<img width="1244" alt="Screen Shot 2020-06-25 at 9 59 08 PM" src="https://user-images.githubusercontent.com/5577409/85812636-2cbab180-b72f-11ea-861e-4e67c695b9a3.png">

However when I try to go back to this page with a refresh, or by entering the team directly in the url I get a 404 error page.
<img width="1244" alt="Screen Shot 2020-06-25 at 9 59 17 PM" src="https://user-images.githubusercontent.com/5577409/85812695-52e05180-b72f-11ea-8750-2bdea998cb5e.png">

What could be the reason?

**Comment by jamesmosier** [comment, marked as answer]
Hi @ppot! Do you happen to have any more code you could share? I'm not able to reproduce this with a minimal setup as you described.

**Reply by ppot** [reply]
So..... Trying to make a project to share the code with you, i fond that the new project worked. 

The thing that changed is the fact that i rebuild my whole image and container. So I think that, the page [team].tsx was not resolved in the container at that moment. After the rebuild, the route worked. I feel like i need rebuild my image for that matter on the feature. Thanks a lot. :)
